pygments/lexers/yara.py

```diff
@@ -0,0 +1,56 @@
+"""
+    pygments.lexers.yara
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for YARA.
+
+    :copyright: Copyright 2006-2023 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+from pygments.lexer import RegexLexer, words
+from pygments.token import Comment, String, Name, Text, Punctuation, Operator, Keyword, Whitespace, Number, Generic
+
+__all__ = ['YaraLexer']
+
+class YaraLexer(RegexLexer):
+    """
+    For YARA rules
+
+    .. versionadded:: 2.16.0
+    """
+
+    name = 'YARA'
+    url = 'https://virustotal.github.io/yara/'
+    aliases = ['yara', 'yar']
+    filenames = ['*.yar']
+    mimetypes = ['text/x-yara']
+
+    tokens = {
+        'root': [
+            (r'\s+', Whitespace),
+            (r'//.*?$', Comment.Single),
+            (r'\#.*?$', Comment.Single),
+            (r'/\*', Comment.Multiline, 'comment'),
+            (words(('rule', 'private', 'global', 'import', 'include'), prefix=r'\b', suffix=r'\b'), Keyword.Declaration),
+            (words(('strings', 'condition', 'meta'), prefix=r'\b', suffix=r'\b'), Keyword),
+            (words(('ascii', 'at', 'base64', 'base64wide', 'condition', 'contains', 'endswith', 'entrypoint', 'filesize', 'for', 'fullword', 'icontains', 'iendswith', 'iequals', 'in', 'include', 'int16', 'int16be', 'int32', 'int32be', 'int8', 'int8be', 'istartswith', 'matches', 'meta', 'nocase', 'none', 'of', 'startswith', 'strings', 'them', 'uint16', 'uint16be', 'uint32', 'uint32be', 'uint8', 'uint8be', 'wide', 'xor', 'defined'), prefix=r'\b', suffix=r'\b'), Generic.Strong),
```

Comment:
I am unsure what to use here. Those keywords should stand out in rules, since they are at the end of strings. But I didnt want them to have the same format as "Keyword". Any suggestions?

Reply:
``Name.Builtin`` seems good, updated with 48c0680769a23aaeba0c87f221a102d660ee3e74